notebooks/concepts/Specviz from splot concept notebook.ipynb

```diff
@@ -0,0 +1,3636 @@
+    "There should be a redshift slider (optional?) somewhere in the specviz UI.  This slider should update the redshift/radial_velocity recorded in the spectrum's `.spectral_axis.redshift`/.spectral_axis.radial_velocity` attributes.\n",
+    "\n",
+    "1. In the notebook these can be accessed directly as informational items:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "specviz.current_spectrum.redshift, specviz.current_spectrum.radial_velocity"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "    Similarly, the functionality of the `u` and `v` keys are also available in the notebook as long as the pixel location under the cursor is available - e.g., to do the shift that corresponds to `u` followed by `z` and `6563` as the rest wavelength:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "obs_wl = specviz.get_spectrum_location()[0]  #<- blocks until the user clicks a line\n",
+    "specviz.current_spectrum.redshift = obs_wl/6563-1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Similar operations on `specviz.current_spectrum.spectral_axis` allow `v` or other more complex workflows."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2. A plugin could be written to handle workflows like `u` and `v` below and have them set the redshift appropriately.  It is not clear these are common enough to need to be in a plugin as opposed to directing users to the notebook as stated above, however."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Line lists "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "While the redshift is useful contextual information, it is most important when used to identify spectral lines.  For this purpose the user needs to be able to input line lists appropriate for their science case. The below describe how to specify such line lists, but regardless the intent is for the user to then see those lines displayed in `specviz`, redshifted to wherver they should be given the value of the `.redshift`/`.radial_velocity` attribute.\n",
+    "\n",
+    "1. In the notebook line lists should be loadable straightforwardly as astropy tables with spectrally-appropriate units.  The most straightforward option is to consume any table that has 'linename' and 'rest' columns:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "linelist_table = astropy.table.Table()\n",
+    "linelist_table['linename'] = ['Halpha', 'Hbeta']\n",
+    "linelist_table['rest'] = [6563, 4862]*u.angstrom\n",
+    "linelist_table['more information'] = ...\n",
+    "\n",
+    "specviz.load_linelist(linelist_table)"
+   ]
```

Comment:
Loading and displaying line lists is a crucial, high-priority functionality.   Creating observed line lists by clicking on peaks, troughs and auto-finding the lines would also be a very useful capability.

Reply:
Agreed this is important, I'm not sure it needs to be done *in jdaviz*.

But either way I agree the clicking might be useful.  Will add a short section about this.